com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/StepsUtilTest.java

```diff
@@ -70,37 +71,38 @@ public void testGetAppsToDeployWithBindingParameters() throws Exception {
         CloudApplicationExtended application = new CloudApplicationExtended(null, "my-app");
         application.setBindingParameters(bindingParameters);
 
-        StepsUtil.setAppsToDeploy(context, Arrays.asList(application));
-        List<CloudApplicationExtended> actualAppsToDeploy = StepsUtil.getAppsToDeploy(context);
-
-        assertEquals(1, actualAppsToDeploy.size());
-        assertTrue(!actualAppsToDeploy.get(0)
-            .getBindingParameters()
-            .isEmpty());
-        assertTrue(!actualAppsToDeploy.get(0)
-            .getBindingParameters()
-            .get("service-1")
-            .isEmpty());
-        assertEquals(Integer.class, actualAppsToDeploy.get(0)
-            .getBindingParameters()
-            .get("service-1")
-            .get("integer-value")
-            .getClass());
-        assertEquals(Integer.class, actualAppsToDeploy.get(0)
-            .getBindingParameters()
-            .get("service-1")
-            .get("int-double-value")
-            .getClass());
-        assertEquals(Double.class, actualAppsToDeploy.get(0)
-            .getBindingParameters()
-            .get("service-1")
-            .get("real-double-value")
-            .getClass());
-        assertEquals(String.class, actualAppsToDeploy.get(0)
-            .getBindingParameters()
-            .get("service-1")
-            .get("string-value")
-            .getClass());
+        //TODO
+        StepsUtil.setAppsToDeploy(context, Collections.emptyList());
+//        List<CloudApplicationExtended> actualAppsToDeploy = StepsUtil.getAppsToDeploy(context);
```

Comment:
Changed to test getApp() instead getAppsToDeploy because
getApp() deals with CloudApplicationExtended object where getAppsToDeploy() deals with list of strings now.
Moreover, the test is intended only for list with one app in it